frontend/src/metabase/dashboard/components/DashboardHeader/DashboardHeader.tsx

```diff
@@ -291,6 +295,12 @@ class DashboardHeaderContainer extends Component<DashboardHeaderProps> {
   }
 
   getEditingButtons() {
+    const { missingRequiredParameters } = this.props;
```

Comment:
now you desctructure variables from props 😄

Reply:
Yeah, when it's too long, it becomes worse to read `props.missingRequiredParameters`.
What I'd do instead is make a reference to `const p = this.props` and use `p.myParameter` but I believe one-letter variable names aren't favorable by the team.